engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -234,6 +237,8 @@ class WorkflowActor(workflowToStart: WorkflowToStart,
 
   private val startTime = System.currentTimeMillis()
 
+  private val DeleteWorkflowFiles = conf.getBoolean("system.delete-workflow-files")
```

Comment:
Does this `val` start with the uppercase letter for a reason or it is a typo?

Reply:
🤷‍♂ I've changed it, but there was already a `deleteWorkflowFiles` function, so that will now be named `def deleteFiles()`.